front/src/modules/ui/layout/navbar/NavWorkspaceButton.tsx

```diff
@@ -47,17 +48,23 @@ const StyledName = styled.div`
 `;
 
 function NavWorkspaceButton() {
+  const navigate = useNavigate();
+
   const currentUser = useRecoilValue(currentUserState);
 
   const currentWorkspace = currentUser?.workspaceMember?.workspace;
 
+  const onWorksapceClick = () => {
+    navigate('/');
```

Comment:
I think design wants to completely remove the cursor pointer and have no redirection to "/" as this page is actually redirecting to "/people"